test_sets/Defects/Defects_release_7_1.txt

```diff
@@ -40,3 +40,5 @@
 ./test_scripts/Defects/7_1/3620/3620_3_RPC_Service_NACK_non_existed_session_id.lua
 ./test_scripts/Defects/7_1/3620/3620_4_RPC_Service_NACK_protected_non_existed_session_id.lua
 ./test_scripts/Defects/7_1/3620/3620_5_RPC_Service_NACK_before_OnSystemTimeReady.lua
+./test_scripts/Defects/7_1/3556_1_GetListOfPermission_no_specific_permission.lua
+./test_scripts/Defects/7_1/3556_2_GetListOfPermission_contains_groups_with_user_consent_prompt.lua
```

Comment:
Since this fix wasn't added until after the 7.1 release, I would move this to a new "7_2" folder and test set

Reply:
@jacobkeeler Please find update in 9282da2